ilastik/applets/pixelClassification/pixelClassificationDataExportGui.py

```diff
@@ -48,8 +41,15 @@ def __init__(self, *args, **kwargs):
         super(PixelClassificationResultsViewer, self).__init__(*args, **kwargs)
         self.topLevelOperatorView.PmapColors.notifyDirty(bind(self.updateAllLayers))
         self.topLevelOperatorView.LabelNames.notifyDirty(bind(self.updateAllLayers))
+        self._childOps = []
```

Comment:
looks like there will always be only one of those `childops`? Why would it be a list?